prov/efa/test/efa_unit_test_common.c

```diff
@@ -153,3 +154,28 @@ void efa_unit_test_buff_destruct(struct efa_unit_test_buff *buff)
 
 	free(buff->buff);
 }
+
+/**
+ * @brief Construct RXR_EAGER_MSGRTM_PKT
+ *
+ * @param[in] pkt_entry Packet entry. Must be non-NULL.
+ * @param[in] attr Packet attributes.
+ */
+void efa_unit_test_eager_msgrtm_pkt_construct(struct rxr_pkt_entry *pkt_entry, struct efa_unit_test_eager_rtm_pkt_attr *attr)
```

Comment:
I think we can refactor this function in the future to support other packet types.